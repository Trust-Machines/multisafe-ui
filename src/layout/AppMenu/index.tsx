import React from 'react';
import {Box} from '@mui/material';
import useMediaBreakPoint from "../../hooks/useMediaBreakPoint";

const AppMenu = (props: { children?: React.ReactNode }) => {
    const [isMd] = useMediaBreakPoint();

    const boxStyles = {
        width: isMd ? '200px' : '100%',
        flexGrow: 0,
        flexShrink: 0
    }

    return <Box sx={boxStyles}>{props.children}</Box>
}

export default AppMenu;