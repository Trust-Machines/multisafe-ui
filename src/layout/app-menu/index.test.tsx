import {render} from '@testing-library/react';

import AppMenu from './index';

test('1 Default render', () => {
    const rendered = render(<AppMenu><span>children</span></AppMenu>);
    expect(rendered.container).toMatchSnapshot()
});