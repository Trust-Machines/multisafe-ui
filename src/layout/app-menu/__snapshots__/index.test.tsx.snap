// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1 Default render 1`] = `
<div>
  <div
    class="MuiBox-root css-cumda7"
  >
    <span>
      children
    </span>
  </div>
</div>
`;
